docs/root/configuration/listeners/runtime.rst

```diff
@@ -4,5 +4,5 @@ Runtime
 Listeners support the following runtime settings:
 
 ssl.alt_alpn
-  What % of requests use the configured :ref:`alt_alpn <config_listener_ssl_context_alt_alpn>`
+  **Deprecated in v2.** What % of requests use the configured :ref:`alt_alpn <config_listener_ssl_context_alt_alpn>`
```

Comment:
Other docs (e.g. `docs/root/operations/cli.rst`) use `*(deprecated)*`, so maybe stick to that?

Looks good, otherwise.

Reply:
Right, I just didn't realize that. Thanks.